doc/nrf/releases_and_maturity/releases/release-notes-2.6.1.rst

```diff
@@ -0,0 +1,115 @@
+
+Release notes might refer to "experimental" support for features, which indicates that the feature is incomplete in functionality or verification, and can be expected to change in future releases.
+To learn more, see :ref:`software_maturity`.
+
+Highlights
+**********
+
+This patch release adds changes on top of the :ref:`nRF Connect SDK v2.6.0 <ncs_release_notes_260>`.
+
+See :ref:`ncs_release_notes_261_changelog` for the complete list of changes.
+
+Release tag
+***********
+
+The release tag for the |NCS| manifest repository (|ncs_repo|) is **v2.6.1**.
+Check the :file:`west.yml` file for the corresponding tags in the project repositories.
+
+To use this release, check out the tag in the manifest repository and run ``west update``.
+See :ref:`cloning_the_repositories` and :ref:`gs_updating_repos_examples` for more information.
+
+For information on the included repositories and revisions, see `Repositories and revisions for v2.6.1`_.
+
+IDE and tool support
+********************
+
+`nRF Connect extension for Visual Studio Code <nRF Connect for Visual Studio Code_>`_ is the recommended IDE for |NCS| v2.6.1.
+See the :ref:`installation` section for more information about supported operating systems and toolchain.
+
+Supported modem firmware
+************************
+
+See `Modem firmware compatibility matrix`_ for an overview of which modem firmware versions have been tested with this version of the |NCS|.
+
+Use the latest version of the nRF Programmer app of `nRF Connect for Desktop`_ to update the modem firmware.
+See :ref:`nrf9160_gs_updating_fw_modem` for instructions.
+
+Modem-related libraries and versions
+====================================
+
+.. list-table:: Modem-related libraries and versions
+   :widths: 15 10
+   :header-rows: 1
+
+   * - Library name
+     - Version information
+   * - Modem library
+     - `Changelog <Modem library changelog for v2.6.1_>`_
+   * - LwM2M carrier library
+     - `Changelog <LwM2M carrier library changelog for v2.6.1_>`_
+
+Known issues
+************
+
+Known issues are only tracked for the latest official release.
+See `known issues for nRF Connect SDK v2.6.1`_ for the list of issues valid for the latest release.
+
+.. _ncs_release_notes_261_changelog:
+
+Changelog
+*********
+
+The following sections provide detailed lists of changes by component.
+
+Protocols
+=========
+
+This section provides detailed lists of changes by :ref:`protocol <protocols>`.
+
+Matter
+------
+
+* Updated:
+
+  * The default Message Reliability Protocol (MRP) retry intervals for Thread devices to two seconds to reduce the number of spurious retransmissions in Thread networks.
+  * Increased the number of available packet buffers in the Matter stack to avoid packet allocation issues.
+
+Thread
+------
+
+* Fixed ``otPlatCryptoPbkdf2GenerateKey`` API implementation to allow a fallback to legacy MbedTLS implementation instead of returning ``OT_ERROR_NOT_CAPABLE``.
```

Comment:
```suggestion
* Fixed ``otPlatCryptoPbkdf2GenerateKey`` API implementation to allow a fallback to legacy Mbed TLS implementation instead of returning ``OT_ERROR_NOT_CAPABLE``.
```